### DISCUSSION TITLE
PR Getting blocked because app removed from vercel still has "required" status

### DISCUSSION THREAD

**Discussion by slaterhaus** [vercel/community#1617] [Help] [answered]
### Summary

I have removed a deprecated application from my vercel dashboard, but in my pull requests, the application status hangs because it will no longer build. This wouldn't be a problem except that the status is marked as "required" in order for a PR to be merged. As a result, it is now blocking all PR merging in the repo. What can I do about this? Would be optimal to be able to remove the step entirely, but at the very least, I would like to be able to skip the status check.

**Comment by amyegan** [comment, marked as answer]
Hi James! I think you're talking about GitHub rules blocking a PR rather than Vercel deployments being blocked, so I'm going to answer based on that assumption. Please let me know if I misunderstood, and I will get you the right info.

You might need to change your branch protection rules to remove a check that is no longer needed. You can find a list of branches with protection rules at `github.com/<user-name>/<project-name>/settings/branches`. From there you can click "Edit" to change a branch's protection rules and remove any checks that should no longer be required

**Reply by slaterhaus** [reply]
Thank you, I didn't have admin access to my repo so I definitely missed it!
